build.gradle.kts

```diff
@@ -720,7 +719,7 @@ tasks {
 
     register<Download>("downloadHelmDocsArchive") {
         description = "Download helm-docs release archive"
-        group = "build"
+        group = "build setup"
```

Comment:
Is this a default Gradle group? Otherwise maybe just name the group `buildHelmDocs` or something like that?

Reply:
Yep. That's internal group that already contains a few tasks:
```shell
./gradlew tasks

Build Setup tasks
-----------------
init - Initializes a new Gradle build.
resolveSwaggerTemplate - Resolves template files from the swaggerTemplate configuration.
updateDaemonJvm - Generates or updates the Gradle Daemon JVM criteria.
wrapper - Generates Gradle wrapper files.
```